java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Add ssm text to checkbox table header on system list pages that don't have add to ssm button
```

Comment:
I'd better rephrase this to be clear to users.
Instead of
`.Add ssm text to checkbox table header on system list pages that don't have add to ssm button`
replace it with
`Clarify the behavior of the checkbox system list, when it adds systems to ssm`

Reply:
ok. i'll do that. thanks.